compiler/rustc_target/src/spec/targets/riscv64_wrs_vxworks.rs

```diff
@@ -0,0 +1,24 @@
+use crate::spec::{base, StackProbeType, Target, TargetOptions};
+
+pub(crate) fn target() -> Target {
+    Target {
+        llvm_target: "riscv64".into(),
+        metadata: crate::spec::TargetMetadata {
+            description: None,
+            tier: Some(3),
+            host_tools: Some(false),
+            std: Some(true),
+        },
+        pointer_width: 64,
+        data_layout: "e-m:e-p:64:64-i64:64-i128:128-n32:64-S128".into(),
+        arch: "riscv64".into(),
+        options: TargetOptions {
+            cpu: "generic-rv64".into(),
+            llvm_abiname: "lp64d".into(),
+            max_atomic_width: Some(64),
```

Comment:
According to [TargetOptions::max_amonic_width](https://doc.rust-lang.org/nightly/nightly-rustc/rustc_target/spec/struct.TargetOptions.html#structfield.max_atomic_width):

> max_atomic_width: [Option](https://doc.rust-lang.org/nightly/core/option/enum.Option.html)<[u64](https://doc.rust-lang.org/nightly/std/primitive.u64.html)>
Don’t use this field; instead use the .max_atomic_width() method.

I think you are supposed to use the `max_atomic_width()` method instead

Reply:
i think that's about accessing it, not setting it

Reply:
I see. We can resolve this comment.